docs/_ug/KeyDefinitions.md

```diff
@@ -32,7 +32,7 @@ Feel free to add tags as you see fit to organize your inventory. Examples of how
 
 Tags can be renamed and these changes would be reflected on all items immediately.
 
-FoodRem Tags are unique by name and case-sensitive. This means you cannot add two or more tags of the same name.
+FoodRem Tags are unique by name and case-insensitive. This means you cannot add two or more tags of the same name.
```

Comment:
They are case sensitive. "Potato" and "potato" are different. Right? @Eugenetanwl3881

Reply:
Right now, yes. Unless we planning to change